.travis.yml

```diff
@@ -30,19 +28,25 @@ matrix:
       env: SYMFONY_VERSION=4.1.*;ASSETIC=skip
     - php: 7.2
       env: SYMFONY_VERSION=@dev;ASSETIC=skip
+    - php: 7.3
+    env: SYMFONY_VERSION=3.3.*
+    - php: 7.3
+      env: SYMFONY_VERSION=3.4.*
+    - php: 7.3
+      env: SYMFONY_VERSION=4.0.*;ASSETIC=skip
+    - php: 7.3
+      env: SYMFONY_VERSION=4.1.*;ASSETIC=skip
+    - php: 7.3
+      env: SYMFONY_VERSION=@dev;ASSETIC=skip
     - php: 7.0
-      env: TWIG_VERSION=2.x
-    - php: 5.6
-      env: COMPOSER_FLAGS="--prefer-lowest"
+      env: TWIG_VERSION=2.7.*
```

Comment:
why `2.7.*` ? That will be unmaintained as soon as Twig 2.8 is released.

Reply:
I think the intention is to test the lower bound, but I guess then it should be 2.4